README.md

```diff
@@ -63,7 +63,7 @@ SentrySDK.start(options: [
 
 For more information checkout the [docs](https://docs.sentry.io/platforms/cocoa/?platform=swift).
 
-<sup>(1)</sup>limited symbolication support
+<sup>(1)</sup>limited symbolication support and no crash handling.
```

Comment:
M: what's about raising a PR for https://docs.sentry.io/platforms/cocoa/?platform=swift#install too? I'm all in favor of a single source of truth, but this is very important.
this comment doesn't block this PR btw.

Reply:
Good point, I'm going to change this as well.

Reply:
https://github.com/getsentry/sentry-docs/pull/1924